code/modules/holodeck/turfs.dm

```diff
@@ -3,16 +3,17 @@
 	thermal_conductivity = 0
 	broken_states = list("engine")
 	burnt_states = list("engine")
+	flags = NONE
 
-/turf/open/floor/holofloor/attackby(obj/item/weapon/W as obj, mob/user as mob)
+/turf/open/floor/holofloor/attackby()
```

Comment:
Please keep the signature the same even if the parameters are not used. In case they're used as named arguments anywhere.